cprover_bindings/src/goto_program/location.rs

```diff
@@ -46,18 +55,49 @@ impl Location {
         }
     }
 
+    pub fn function_name(&self) -> Option<InternedString> {
+        match self {
+            Location::Loc { function, .. } => *function,
+            _ => None,
+        }
+    }
+
+    pub fn get_column_number(&self) -> Option<u64> {
+        match self {
+            Location::Loc { col, .. } => *col,
+            _ => None,
+        }
+    }
+
+    pub fn get_comment(&self) -> Option<InternedString> {
+        match self {
+            Location::Property { comment, .. } => *comment,
+            _ => None,
+        }
+    }
+
+    pub fn get_property_class(&self) -> Option<InternedString> {
+        match self {
+            Location::Property { property_class, .. } => *property_class,
+            _ => None,
+        }
+    }
+
     /// Convert a location to a short string suitable for (e.g.) logging.
     /// Goal is to return just "file:line" as clearly as possible.
     pub fn short_string(&self) -> String {
         match self {
             Location::None => "<none>".to_string(),
-            Location::BuiltinFunction { function_name, line: Some(line) } => {
+            Location::BuiltinFunction { function_name, line: Some(line), .. } => {
```

Comment:
There are multiple locations in the PR where you're adding dots to patterns, and these appear to be remnants of a previous version of the PR. Please fix them all.

Reply:
Yup, thanks for spotting these.